ui/components/component-library/base-avatar/base-avatar.scss

```diff
@@ -23,6 +23,8 @@
 
   height: var(--avatar-size);
   width: var(--avatar-size);
+  max-width: var(--avatar-size);
+  flex: 0 0 var(--avatar-size);
   border-radius: 100%;
```

Comment:
Will move this to a separate PR it fixes the display of the Avatar in a flex-box context with elements of different heights.

### Before

<img width="418" alt="Screen Shot 2022-09-30 at 11 54 12 AM" src="https://user-images.githubusercontent.com/8112138/193337939-c75a820d-d7a7-4ddf-80b3-f6a5db829add.png">


### After

<img width="368" alt="Screen Shot 2022-09-30 at 11 54 39 AM" src="https://user-images.githubusercontent.com/8112138/193337979-dc277261-e0d2-475f-92e3-c8dc6925a1d9.png">

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/16047